### PR TITLE
Blog post about launching intern program

### PR DESCRIPTION
**Pull request by gvwilson**
Please add comments or Slack Greg Wilson with fixes. Launch date is Mon Feb 24.

### CODE REVIEW

File: content/blog/2020-02-24-applications-for-2020-intern-program-are-now-open/index.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+<div align="center">
+<a href="https://forms.gle/Hhj5WKYp9rC43TnE7"><i class="fas fa-hand-point-right"></i>&nbsp;Apply here</a>
+</div>
+
+## Projects
+
+This year's internships will be divided between our open source and education teams,
+and the projects will be selected from:
+
+1.  **Build a portal for people working with spreadsheets in R**.
```
Comment (jennybryan) [inline content/blog/2020-02-24-applications-for-2020-intern-program-are-now-open/index.Rmarkdown]:
```suggestion
1.  **Create resources for people working with spreadsheets in R**.
```

**Comment by jennybryan** [comment]
Specifics about the spreadsheets project: Primary intern tasks will be writing, synthesis, comparison, exposition, and exampling. This project is not explicitly about package development, although the work could easily lead to pull requests to spreadsheet reading/writing packages. I will look for evidence of general R experience, basic competence with Git/GitHub, previous use of R Markdown, and ability to write clearly about code.

**Comment by mine-cetinkaya-rundel** [comment]
It seems like we're not adding too much detail on expected skills (like what Jenny described above)? If so, LGTM!

**Comment by gvwilson** [comment]
Thanks for the proofread:

1. Why remove `https` from links? (These days I'd probably convert insecure links to HTTPS...)
2. More details on technical skills required from applicants (similar to what's in Max's writeup) would be a help.

**Comment by mine-cetinkaya-rundel** [comment]
I didn't remove them per se, they're just hyperlinked now so that you don't see them when reading. Makes the link shorter, more memorable. If you object, feel free to convert back.

**Comment by mine-cetinkaya-rundel** [comment]
More detail based on Jenny's description has been added.

File: content/blog/2020-02-24-applications-for-2020-intern-program-are-now-open/index.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+tags: internship
+description: |
+  Want to apply to this year's summer intern program? Here's how.
+---
+
+As [we announced last year][preview],
+RStudio's summer intern program will run for a third time in 2020,
+and applications are now open.
+Five people will work with an RStudio mentor full-time for 10 weeks starting in May 2020.
+Everyone is welcome to apply:
```
Comment (topepo) [inline content/blog/2020-02-24-applications-for-2020-intern-program-are-now-open/index.Rmarkdown]: We should comment that the interns should be prepared to effectively work remotely. From previous announcement: 

> RStudio is a geographically distributed team which means you can be based anywhere in the United States (we hope to expand the program to support interns in other countries next year). This means that unless you are based in Boston or Seattle, you will be working 100% remotely, though you will meet with your mentor regularly online, and we will pay for you to travel to one face-to-face work sprint with them.

In the past, we have also stressed that the need to be able to _demonstrate_ that they can effectively work in GitHub: 

> To demonstrate these skills, your application needs to include a link to a package, Shiny app, or data analysis repository on GitHub. It’s OK if you create something specifically for this application: we just want to know that you’re already familiar with the mechanics of collaborative development in R.

I would say a good number of applicants in the past two years had little to no GH projects or activity. We should dissuade these folks from applying.

Comment (gvwilson) [reply content/blog/2020-02-24-applications-for-2020-intern-program-are-now-open/index.Rmarkdown]: @topepo Thanks for the reminder about remote work - paragraph added. The application form asks them about Git repos - in discussion last Friday we decided not to make it an absolute requirement.

**Pull request merged**